docs/understand/roadmap.md

```diff
@@ -14,135 +14,145 @@ redirect_from: ../roadmap.html
 {% include toc.html %}
 
 ---
+## Ecosystem
 
-## Use Case: Development
+### Snowflake Support: External tables on lakeFS <span>Requires Discussion</span>{: .label .label-yellow }
 
-### Ephemeral branches with a TTL
-Throwaway development or experimentation branches that live for a pre-configured amount of time, and are cleaned up afterwards. This is especially useful when running automated tests or when experimenting with new technologies, code or algorithms. We want to see what the outcome looks like, but don’t really need the output to live much longer than the duration of the experiment.
+Since Snowflake supports reading external tables from an object store, we'd like to extend this support to work with lakeFS repositories thaet are hosted on top of [supported object stores](https://docs.snowflake.com/en/user-guide/data-load-snowpipe-intro.html#supported-cloud-storage-services){: target="_blank" }.
```

Comment:
```suggestion
Since Snowflake supports reading external tables from an object store, we'd like to extend this support to work with lakeFS repositories that are hosted on top of [supported object stores](https://docs.snowflake.com/en/user-guide/data-load-snowpipe-intro.html#supported-cloud-storage-services){: target="_blank" }.
```